packages/block-editor/src/components/list-view/branch.js

```diff
@@ -105,12 +105,17 @@ function ListViewBranch( props ) {
 	const parentBlockInformation = useBlockDisplayInformation( parentId );
 	const syncedBranch = isSyncedBranch || !! parentBlockInformation?.isSynced;
 
-	const canParentExpand = useSelect(
+	const { canParentExpand, getBlockById } = useSelect(
```

Comment:
List View also can't be accessed in zoomed out mode now, so the changes here might also not be needed